content/patterns/alert/examples/alert.html

```diff
@@ -20,85 +20,84 @@
         <div id="ex1">
-          <button type="button" id="alert-trigger">Trigger Alert</button>
+          <button type="button" id="alert-trigger">アラートをトリガーする</button>
 
           <div id="example" role="alert"></div>
 
-          <!--  The following script element contains the content that will be inserted into the alert element. -->
+          <!--  以下のscript要素には、アラート要素に挿入される内容が含まれています。 -->
           <script type="text/template" id="alert-template">
-            <p>Hello</p>
+            <p>こんにちは</p>
           </script>
         </div>
-        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_label ex_end_sep" aria-label="の終了"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティの特徴</h2>
         <p>
-          Because an alert is for critical information, assistive technologies may provide special behaviors designed to help call attention to changes in the text of an alert.
-          For example, screen readers may interrupt all other speech and preface announcement of the new alert text with a special sound or phrase.
+          アラートは重要な情報のため、支援技術はアラートのテキストの変更に注意を向けるための特別な動作を提供する場合があります。例えば、スクリーンリーダーは他のすべての読み上げを中断し、新しいアラートテキストの前に特別な音又はフレーズでアナウンスする場合があります。
         </p>
       </section>
 
       <section>
-        <h2 id="kbd_label">Keyboard Support</h2>
-        <p>No keyboard interaction needed.</p>
+        <h2 id="kbd_label">キーボードのサポート</h2>
+        <p>キーボードによるインタラクションは必要ありません。</p>
       </section>
 
       <section>
-        <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
+        <h2 id="rps_label">ロール、プロパティ、ステート、及び tabindex 属性</h2>
         <table aria-labelledby="rps_label" class="data attributes">
           <thead>
             <tr>
-              <th scope="col">Role</th>
-              <th scope="col">Attribute</th>
-              <th scope="col">Element</th>
-              <th scope="col">Usage</th>
+              <th scope="col">ロール</th>
+              <th scope="col">属性</th>
+              <th scope="col">要素</th>
+              <th scope="col">使用法</th>
             </tr>
           </thead>
           <tbody>
             <tr data-test-id="alert-role">
               <th scope="row"><code>alert</code></th>
               <td></td>
               <td><code>div</code></td>
-              <td>Identifies the element as the container where alert content will be added or updated.</td>
+              <td>アラートのコンテンツが追加又は更新されるコンテナとしての要素を識別します。</td>
             </tr>
             <tr data-test-id="test-not-required">
               <td></td>
               <th scope="row">
                 <code>aria-live=<q>assertive</q></code>
               </th>
-              <td>Implicit on <code>div</code></td>
+              <td><code>div</code> に暗黙的</td>
```

Comment:
「divに暗黙的」はだいぶわかりにくい感じがしますね……
対象要素はdivであり、そのdivにaria-live属性が暗黙的に指定された状態になる、という話ですけれども。
少し補ってこう?

```suggestion
              <td><code>div</code> に暗黙的に割り当て</td>
```

Reply:
自分でも訳していて、「divに暗黙的」の意味するところがわからなかったのですが、なるほど、すごく腹落ちしました！